crates/cxx-qt-gen/src/naming/name.rs

```diff
@@ -237,6 +237,25 @@ impl Name {
         self.namespace.as_deref()
     }
 
+    pub fn module_ident(&self) -> Option<&Ident> {
+        if let Some(path) = &self.module {
+            path.get_ident()
+        } else {
+            None
+        }
+    }
+
+    pub fn require_module_ident(&self) -> Result<&Ident> {
+        if let Some(ident) = self.module_ident() {
+            Ok(ident)
+        } else {
+            Err(Error::new_spanned(
+                self.rust_unqualified(),
+                format!("No Module name for {}!", self.rust_unqualified()),
+            ))
+        }
+    }
```

Comment:
some docs lines on the two methods would be good :-)

Reply:
Done